comps/llms/text-generation/bedrock/entrypoint.sh

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
```

Comment:
Since all this file is doing is to just a python call, I'd delete file and update the last line of the `comps/llms/text-generation/bedrock/Dockerfile` this way:
```
ENTRYPOINT ["python", "llm.py"]
```